### DISCUSSION TITLE
Project still alive?

### DISCUSSION THREAD

**Discussion by pgtaboada** [vsch/flexmark-java#467] [General]
Just wondering...

brgds

Papick

**Comment by vsch** [comment]
Yes it is still alive but with clarification. 

I took on a four to six week project in February which morphed into a six month, 7 days a week black hole. I had no time at all to work on flexmark or anything else.

I am now starting another two month urgent project after which I am going to take a break so I can update my personal projects, including flexmark.

I read all the issues and try to respond to ones which have not been addressed before.

I am planning to review and merge accumulated pull requests before embarking on the two month project.

**Comment by vsch** [comment]
@pgtaboada, best laid plans of mice and men. I wound up taking 2 years off, no computer, no programming. I am now slowly getting back into it. I planning to address accumulated PRs and issues in the next few weeks.

I am addressing a backlog of issues on my IntelliJ plugins, which I myself need for development. Once that is cleared out I will be able to address `flexmark-java`.

**Comment by david-waltermire** [comment]
@vsch I hope you are well. Any status update? Many of us really appreciate this library and would love to see maintenance continue.

**Comment by bennadel** [comment]
I just wanted to drop by and say that I love this project and it has been so helpful for my projects. Just wanted to send some love - I know that being a sole-maintainer is an overwhelming job, especially in the modern AI world where things all just feel... crazy. We appreciate everything you do!
